actors/builtin/power/policy.go

```diff
@@ -4,11 +4,20 @@ import (
 	abi "github.com/filecoin-project/specs-actors/actors/abi"
 )
 
-// Minimum number of registered miners for the minimum miner size limit to effectively limit consensus power.
-const ConsensusMinerMinMiners = 3
+// PARAM_SPEC
+// This is a switch for turning on the minimum miner size requirement.
+// When this number of miners of minimum miner size is reached, minimum miner size is enforce.
+const ConsensusMinerMinMiners = 4
 
-// Minimum power of an individual miner to meet the threshold for leader election.
-var ConsensusMinerMinPower = abi.NewStoragePower(1 << 40) // PARAM_FINISH
+// PARAM_SPEC
+// Minimum miner size, the minimum power of an individual miner to meet the threshold for leader election (in bytes).
+// Motivation:
+// - Limits sybil generation
+// - Improves consensus fault detection
+// - Guarantees a minimum fee for consensus faults
+// - Ensures that a specific soundness for the power table
+// Future: we can consensus fault fee and sybil generation with crypto econ mechanic and we can mantain the target soundness by increasing the challenges for small miners.
+var ConsensusMinerMinPower = abi.NewStoragePower(100 << 40) // PARAM_FINISH
```

Comment:
**Clarification request:**
- This was set to 1TiB (assuming that the value is in bytes), can someone confirm? @anorth 
- We set this to 100TiB, pending more research to lower it down.

Reply:
It was, because politics.